src/pages/workspaces/workspace/workflows/LaunchAnalysisModal.js

```diff
@@ -53,6 +54,7 @@ const LaunchAnalysisModal = ({
       const { submissionId } = await launch({
         isSnapshot: type === processSnapshotTable,
         workspace, config, selectedEntityType, selectedEntityNames, newSetName, useCallCache, deleteIntermediateOutputFiles, useReferenceDisks,
+        memoryRetryMultiplier: retryWithMoreMemory ? retryMemoryFactor : undefined,
```

Comment:
The back end code expects just memoryRetryMultiplier. The default in the back end code is 1.0, which means no increase in memory. In the front end, the minimum allowable value is 1.1, so as long as the checkbox was selected we should send the user-specified value.